token_auth/views/password_reset.py

```diff
@@ -0,0 +1,30 @@
+from django.shortcuts import render, redirect, reverse
+from rest_framework import generics, status, views, permissions
+from token_auth.serializers.password_reset import ResetPasswordEmailRequestSerializer
+from rest_framework.response import Response
+from django.contrib.sites.shortcuts import get_current_site
+from django.contrib.auth.models import User
+from django.contrib.auth.tokens import PasswordResetTokenGenerator
+from django.utils.encoding import smart_str, force_str, smart_bytes, DjangoUnicodeDecodeError
+from django.utils.http import urlsafe_base64_decode, urlsafe_base64_encode
+from token_auth.utils import Util
+
+
+class RequestPasswordResetEmailView(generics.GenericAPIView):
+    serializer_class = ResetPasswordEmailRequestSerializer
+    def post(self, request):
+        serializer = self.serializer_class(data=request.data)
+        email = request.data.get('email', '')
+        if User.objects.filter(email=email).exists():
+            user = User.objects.get(email=email)
+            uidb64 = urlsafe_base64_encode(smart_bytes(user.id))
+            token = PasswordResetTokenGenerator().make_token(user)
+            current_site = get_current_site(request=request).domain
+            relativeLink = reverse('password-reset-confirm', kwargs={'uidb64': uidb64, 'token': token})
+            redirect_url = request.data.get('redirect_url', '')
+            absurl = 'http://'+current_site + relativeLink
+            email_body = 'Hello, \n Use link below to reset your password  \n' + \
+                absurl+"?redirect_url="+redirect_url
```

Comment:
Can you use double quotes in a consistent manner, please? By this I mean, if you use double-quote here, then use it everywhere on this file.

Reply:
Done